lib/xcresult/models.rb

```diff
@@ -420,4 +423,126 @@ def failure_message
       new_message
     end
   end
+
+  # - ActionTestSummary
+  #   * Supertype: ActionTestSummaryIdentifiableObject
+  #   * Kind: object
+  #   * Properties:
+  #     + testStatus: String
+  #     + duration: Double
+  #     + performanceMetrics: [ActionTestPerformanceMetricSummary]
+  #     + failureSummaries: [ActionTestFailureSummary]
+  #     + activitySummaries: [ActionTestActivitySummary]
+  class ActionTestSummary < ActionTestSummaryIdentifiableObject
```

Comment:
> As this is a version of ActionTestSummaryIdentifiableObject, you'll want to add it to ActionTestSummaryIdentifiableObject's create method.

Ah I see, I totally missed this but will update that

> Also, it would probably help to improve this method:

Oh yeah, I see what you are saying... This one might be a little bit beyond me, but I agree that it needs improving


Will take a little look later on today 👍